### PR TITLE
chore: try file-based labeling

### PR DESCRIPTION
**Pull request by ennru**
Refs #551﻿

### CODE REVIEW

**Review by octonato — APPROVED** [review]
Seems that this should work better. 
We may want to also tag changed in testkits, maven/sbt. But we can add later once we confirm this will work as expected.

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -27,9 +27,15 @@ autolabeler:
   - label: 'java-sdk'
     title:
       - '/.*\(.*java.*\):/i'
+    files:
+      - '/sdk/java-sdk/**'
   - label: 'scala-sdk'
     title:
       - '/.*\(.*scala.*\):/i'
```
Comment (patriknw) [inline .github/release-drafter.yml:34]: can we remove the title sections now?

**Comment by patriknw** [comment]
> We may want to also tag changed in testkits, maven/sbt

I don't think we should have too many categories because each PR that touches many files will show up duplicated in all matching sections. I think users will see testkits and build tools as part of the sdk.

**Comment by patriknw** [comment]
You can try by touching some matching file, but I still think there is something missing to make this work. We had file based matching previously but I couldn't see any labels that weren't manually added.

File: .github/release-drafter.yml (hunk 1 above)
Comment (ennru) [reply .github/release-drafter.yml:34]: Let's see if it kicks in with the file globs. Something else might be broken.

**Comment by octonato** [comment]
> I don't think we should have too many categories because each PR that touches many files will show up duplicated in all matching sections. I think users will see testkits and build tools as part of the sdk.

@patriknw, I was not suggesting more categories, but also labeling maven changes with `java-sdk` and sbt change with `scala-sdk`.

**Comment by patriknw** [comment]
That makes total sense, agreed.

**Pull request merged**